presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergSplitManager.java

```diff
@@ -56,9 +58,17 @@ public ConnectorSplitSource getSplits(
             return new FixedSplitSource(ImmutableList.of());
         }
 
+        TupleDomain<IcebergColumnHandle> predicate;
```

Comment:
nit: This would save a couple of lines
```
        TupleDomain<IcebergColumnHandle> predicate = isPushdownFilterEnabled(session)
                ? layoutHandle.getPartitionColumnPredicate().transform(IcebergColumnHandle.class::cast)
                : table.getPredicate();
```

Reply:
Done